tests/findex.test.ts

```diff
@@ -17,12 +17,18 @@ import {
   generateAliases,
   callbacksExamplesBetterSqlite3,
   callbacksExamplesInMemory,
+  toBeBytes,
 } from ".."
 import { USERS } from "./data/users"
 import { expect, test } from "vitest"
 import { createClient, defineScript } from "redis"
 import { randomBytes } from "crypto"
 import Database from "better-sqlite3"
+import * as fs from "fs"
+import * as os from "os"
+
+const FINDEX_TEST_KEY = "6hb1TznoNQFvCWisGWajkA=="
+const FINDEX_TEST_LABEL = "Some Label"
```

Comment:
These key should be stored inside the SQLite database, no?

Reply:
Yes it could. But changing the SQL scheme seems more expensive IMHO.

Reply:
Yes it's only a suggestion for a future improvement, having the key and label inside the non regression SQLite database could simplify the process in the future I think :-)

Reply:
yes agree